_config.yml

```diff
@@ -18,25 +20,25 @@ collections:
     permalink: "/blog/:categories/:year/:month/:day/:title.html"
     
 defaults:
-  - scope:
-      path: "" # an empty string here means all files in the project
-    values:
-      layout: "default"
+  # - scope:
+  #     path: "" # an empty string here means all files in the project
+  #   values:
+  #     layout: "default"
```

Comment:
What is the purpose of commenting this? Consider removing completely, instead of commenting out.